Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/PersonDetailsQrCard.kt

```diff
@@ -68,6 +70,48 @@ class PersonDetailsQrCard(parent: ViewGroup) :
                     )
                 }
             }
+
+            when (certificate.getState()) {
+                is CwaCovidCertificate.State.ExpiringSoon -> {
+                    expirationStatusIcon.visibility = View.VISIBLE
+                    expirationStatusIcon.setImageDrawable(ContextCompat.getDrawable(context, R.drawable.ic_av_timer))
```

Comment:
there is a handy extension for that `context.getDrawableCompat( R.drawable.ic)` , please use it instead of 
`ContextCompat.getDrawable(context, R.drawable.ic)`?